jsapp/js/stores.es6

```diff
@@ -219,52 +219,33 @@ var sessionStore = Reflux.createStore({
   },
   triggerAnonymous (/*data*/) {},
   triggerEnv (environment) {
-    this.environment = environment;
-  },
-  triggerLoggedIn (acct) {
-    this.currentAccount = acct;
-    if (acct.upcoming_downtime) {
```

Comment:
Are we dropping all this upcoming downtime code?

Reply:
Yes :) if there's any more of it, it should be removed. I think it's been broken for a long time and https://github.com/kobotoolbox/kpi/pull/2260 takes care of its job now